src/ipmanager/urls.py

```diff
@@ -14,25 +14,30 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-
 from django.contrib import admin
-from django.urls import path
-
+from django.urls import include, path
 from ipmanager.api.views import CheckView, GroupKeyView, GroupsView
 from ipmanager.ui.views import (CreateGroupView, CreateIPRangeView, CreateRelationView, DeleteGroupView,
-                                DeleteIPRangeView, DeleteRelationView, EditGroupView, GroupListView, HomeView,
+                                DeleteIPRangeView, DeleteRelationView, EditGroupView, GroupListView, HomeView, RootView,
                                 SingleGroupView)
+from djangosaml2.views import AssertionConsumerServiceView, LogoutInitView
 
 urlpatterns = [
-    path('django-admin/', admin.site.urls),
-    path('', HomeView.as_view(), name='home_page'),
-    path('groups/', GroupsView.as_view(), name='groups'),
+    path('', RootView.as_view(), name='site_root'),
+    path('users/auth/saml/callback',
+         AssertionConsumerServiceView.as_view(), name="saml2_acs"),
+    path('users/auth/saml/slo', LogoutInitView.as_view(), name='saml2_logout'),
+    path('saml2/', include('djangosaml2.urls')),
```

Comment:
I think it is fine to leave this in here. I would add a comment explaining why we are manually adding the previous routes at different URLs.